examples/streetlights-operation-security.yml

```
@@ -92,9 +108,9 @@ channels:
         $ref: '#/components/messages/dimLight'
       security:
       # This operation level security implies the ability to send a message to 
```

Isn't this wrong? Publish VS Subscribe Oddisee 😅 

```suggestion
      # This operation level security implies the ability to subscribe to messages from
```